titus-server-master/src/main/java/com/netflix/titus/master/scheduler/TaskPlacementFailureClassifier.java

```diff
@@ -52,7 +52,7 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-class TaskPlacementFailureClassifier {
+class TaskPlacementFailureClassifier<T extends TaskRequest> {
```

Comment:
I think we should use `TitusQueuableTask` as a base type in our code, so it is easier to add new behaviors if needed in the future.

Reply:
that's why it is a generic type, it can be whatever we want. Note in `DefaultSchedulingService` I'm using `TaskPlacementFailureClassifier<V3QueueableTask>`. I think it makes sense to leave the more generic fenzo type only as a lower bound in the generic definition here, because as things are right now, it's the minimum type required for what this class is doing